### PR TITLE
Update de.toml

### PR DESCRIPTION
**Pull request by ghost**
German translation completed

### CODE REVIEW

**Review by gonzalocasas — CHANGES_REQUESTED** [review]
Hi @tr4phix ! Thanks for the PR! Could you please update to use the informal "Du" rather than "Sie"? Thanks!

File: i18n/de.toml [hunk 1]
```diff
@@ -45,41 +45,41 @@ other = "Challenge"
...
 
-Provided by the community"""
+Zur Verf&uuml;gung gestellt von der Community"""
 
 [challenges_taz2_title]
 other = "Bicycle Parking"
 
 [challenges_taz2_description]
-other = """Help us better understand the utilization of bicycle parking facilities.
+other = """Helfen Sie uns, die Nutzung von Fahrradabstellanlagen besser zu verstehen.
```
Comment (gonzalocasas) [inline i18n/de.toml]: still one here

File: i18n/de.toml [hunk 2]
```diff
@@ -45,41 +45,41 @@ other = "Challenge"
...
 
-Provided by Civil Engineering Office"""
+Zur Verf&uuml;gung gestellt vom B&uuml;ro f&uuml;r Bauingenieurwesen"""
 
 [challenges_gsz2_title]
 other = "Awesome Green Spaces"
 
 [challenges_gsz2_description]
-other = """Help us to improve the attractiveness of green spaces by measuring the usage of infrastructure and occupancy of public recreational spaces.
+other = """Helfen Sie uns, die Attraktivit&auml;t von Gr&uuml;nfl&auml;chen zu verbessern, indem Sie die Nutzung der Infrastruktur und die Belegung der &ouml;ffentlichen Erholungsr&auml;ume messen.
```
Comment (gonzalocasas) [inline i18n/de.toml]: and another `Sie` over here

**Review by gonzalocasas — CHANGES_REQUESTED** [review]
there are still a couple of `Sie` in the text

File: i18n/de.toml (hunk 1 above)
Comment (ghost) [reply i18n/de.toml]: Sorry seems not all changes were committed

File: i18n/de.toml [hunk 3]
```diff
@@ -45,41 +45,41 @@ other = "Challenge"
...
 
-Provided by Civil Engineering Office"""
+Zur Verf&uuml;gung gestellt vom B&uuml;ro f&uuml;r Bauingenieurwesen"""
 
 [challenges_ttn_title]
 other = "Breathe!"
 
 [challenges_ttn_description]
-other = """Empower citizens to map and understand air pollution in the city.
+other = """Erm&ouml;gliche den Bewohnern, die Luftverschmutzung in der Stadt zu erfassen und zu verstehen.
```
Comment (loleg) [inline i18n/de.toml:64]: **Bewohner*innen**

Please check if there are any other pronouns that need to be made gender-neutral.

Comment (gonzalocasas) [reply i18n/de.toml:64]: thx, fixed now, and I didn't find others...

**Pull request merged**